integration-test/tests/cms/collaborators.test.ts

```diff
@@ -0,0 +1,11 @@
+import {expect, test} from '@playwright/test';
+test('GET /api/cms/v1/collaborators returns correct data', async ({request}) => {
+const response = await request.get('/api/cms/v1/collaborators');
+expect(response.status()).toBe(200);
```

Comment:
Please could you also adjust the code format.

Reply:
Did the changes,
![image](https://github.com/user-attachments/assets/9a80756f-1fcc-4c11-abff-1e5300d4b210)

Reply:
You need to adjust the format, please compare with other files, also you can use IntelliJ autoformat, please give a try.